extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -117,7 +123,7 @@ export class SkuRecommendationResultsDialog {
 					}
 				}).component();
 				const storageValue = _view.modelBuilder.text().withProps({
-					value: constants.STORAGE_GB(recommendation.targetSku.storageMaxSizeInMb!),
+					value: constants.STORAGE_GB(recommendation.targetSku.storageMaxSizeInMb! / 1024),
```

Comment:
I assume this wouldn't change, but maybe make it as a variable at the top of the page?